lib/flex_deployment_client/requestor/uploaded_file_requestor.rb

```diff
@@ -1,15 +0,0 @@
-module FlexDeploymentClient
-  class UploadedFileRequestor < JsonApiClient::Query::Requestor
-    def create(*)
-      super
-    end
-
-    def dynamic_connection(type, path, params)
-      connection
-    end
-
-    def request(type, path, params)
-      klass.parser.parse(klass, dynamic_connection(type, path, params).run(type, path, params, klass.custom_headers))
-    end
```

Comment:
These two methods do nothing different to the original implementation they override